docs/example/typeinstances.md

```diff
@@ -0,0 +1,79 @@
+# Setup Type Instances with credentials
+
+To use cloud resources like AWS RDS or Google Cloud SQL you need to configure Type Instances which will hold required credentials.
+
+### AWS
+
+1. Create a new User in AWS dashboard [https://console.aws.amazon.com/iam/home?region=eu-west-1#/home](https://console.aws.amazon.com/iam/home?region=eu-west-1#/home)
+
+   - Add the **AdministratorAccess** permissions.
+   - Note the access key and secret key.
+
+1. Create a file with the AWS Credentials:
+
+    ```bash
+    cat > /tmp/aws-ti.yaml << ENDOFFILE
+    typeInstances:
+      - alias: aws-sa
+        attributes:
+          - path: cap.attribute.cloud.provider.aws
+            revision: 0.1.0
+        typeRef:
+          path: cap.type.aws.auth.credentials
+          revision: 0.1.0
+        value:
+          accessKeyID: {ACCESS_KEY}
+          secretAccessKey: {SECRET_KEY}
+    ENDOFFILE
+    ```
+
+1. Create AWS Credentials TypeInstance:
+
+    ```bash
+    export TI_ID=$(capact typeinstance create -f /tmp/aws-ti.yaml -ojson | jq -r '.[].id')
+    ```
```

Comment:
🤔   I think it will be hard to avoid env variables conflicts in docs using the TypeInstance creation. I would recommend reworking this part to:

``````suggestion
1. Create AWS Credentials TypeInstance:

    ```bash
    capact typeinstance create -f /tmp/aws-ti.yaml -ojson
    ```

1. Note the ID.
``````